actions/js-lint/action.yml

```diff
@@ -0,0 +1,30 @@
+# Define action name, description, and inputs
+name: "JS Lint"
+description: "Lint code with Prettier and eslint"
+inputs:
+  working-directory:
+    required: false
+    description: "Working directory"
+    default: "."
```

Comment:
I think working-directory is a standard github action input and we don't need to specify it, do we ?